runtime/jvmti/jvmtiHelpers.cpp

```diff
@@ -909,19 +909,21 @@ getVirtualThreadState(J9VMThread *currentThread, jthread thread)
 				break;
 			case JVMTI_VTHREAD_STATE_PINNED:
 			case JVMTI_VTHREAD_STATE_PARKED:
+				rc = JVMTI_JAVA_LANG_THREAD_STATE_WAITING | JVMTI_THREAD_STATE_PARKED;
```

Comment:
`JVMTI_JAVA_LANG_THREAD_STATE_WAITING` was only introuced in JDK24. Appropriate ifdefs will need to be added. Otherwise, older JDKs will behave incorrectly.

Reply:
Are you thinking of JVMTI_VTHREAD_STATE_WAIT? It looks like `JVMTI_JAVA_LANG_THREAD_STATE_WAITING` has been in the code base for several years.

Reply:
You are right. I read too quickly. i will launch the PR builds